screens/SignUpScreen/SignUpScreen.js

```diff
@@ -1,31 +1,74 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { View, Text, Keyboard, TouchableWithoutFeedback, Button, TextInput } from 'react-native'
 
 
 import color from '../../styles/colors';
 import styles from './styles';
 
 const SignUpScreen = props => {
-    return ( 
-        <TouchableWithoutFeedback onPress={()=>Keyboard.dismiss()}>
-        <View style={styles.container}>
-        <View style={styles.formCard}>
-          <Text style={styles.title}>¡Crea una cuenta!</Text>
-          <View style={styles.input}>
-            <Text style={styles.tag}>Correo electrónico:</Text>
-            <TextInput style={styles.areaInput} placeholder="Ej: 'baldana@uc.cl'" />
-            <Text style={styles.tag}>Contraseña:</Text>
-            <TextInput style={styles.areaInput} />
-            <Text style={styles.tag}>Confirma tu contraseña:</Text>
-            <TextInput style={styles.areaInput} />
-          </View>
+  const [mail, setMail] = useState('')
+  const [password, setPassword] = useState('')
+  const [confirmPassword, setConfirmPassword] = useState('')
+
+  const mailHandler = text => {
+    setMail(text)
+  }
+
+  const passwordHandler = text => {
+    setPassword(text)
+  }
+
+  const confirmPasswordHandler = text => {
+    setConfirmPassword(text)
+  }
+
+  const signUpButtonHandler = () => {
+    console.log(password,confirmPassword,mail)
+    console.log(password === confirmPassword)
+    if (!(password === confirmPassword)) {
+      console.log("las constraseñas no son iguales")
+    } else {
+      console.log("las constraseñas son iguales")
+    }
```

Comment:
Acá solo se imprimen cosas en consola pero no se hace nada con eso, creo que se te olvidó implementar la lógica de la función en sí

Reply:
lo que sucede es que como nos pidieron atomizar las pull-request, toda la funcionalidad del sign-up vendrá en una pull-request independiente a esta que es de sign-in. Es por eso que lo dejé sin terminar.